controls/views.py

```diff
@@ -3048,7 +3048,8 @@ def view_system_assessment_result_summary(request, system_id, sar_id=None):
     project = system.projects.all()[0]
     sar = get_object_or_404(SystemAssessmentResult, pk=sar_id) if sar_id else None
 
-    sar_items = [item for item in sar.assessment_results] if sar.assessment_results != None else []
+    # Get assessment targets results from wrapped SAR data
+    sar_items = [item for item in sar.assessment_results['sar']] if sar.assessment_results != None else []
```

Comment:
```suggestion
    sar_items = [item for item in sar.assessment_results['sar']] if sar.assessment_results != None else []]
```

Reply:
@davidpofo Take a closer look at the first line. The goal is to return an array or and empty array. First line is correct. Suggestion has incorrect syntax.